services/core-api/app/api/mines/mine/models/excemption_fee_status.py

```diff
@@ -0,0 +1,22 @@
+from sqlalchemy.schema import FetchedValue
+from app.extensions import db
+
+from app.api.utils.models_mixins import AuditMixin, Base
+
+class ExemptionFeeStatus(AuditMixin, Base):
+    __tablename__ = 'exemption_fee_status'
+    exemption_fee_status_code = db.Column(db.String, nullable=False, primary_key=True)
+    description = db.Column(db.String(100), nullable=False)
```

Comment:
This size constraint isn't enforced on the DB level, maybe it should be?

Reply:
I actually removed it, we don't want to enforce string lengths at the data level, and this definition is not actually enforced by SQLAlchemy in any way so we have been meaning to just use db.String.